src/duties/autoFix.js

```diff
@@ -1,9 +1,30 @@
 let _ = require('lodash/fp')
-let { execSync } = require('child_process')
+let { execSync, exec } = require('child_process')
 
-let autoFix = async ({ message, warn }) => {
+let run = (command, cb) =>
```

Comment:
if we're wrapping this, lets make this a promise thing so we can await it instead of using callbacks

Reply:
use bluebird promisify